ui/app/templates/components/console/log-json.hbs

```diff
@@ -2,7 +2,10 @@
   <JsonEditor
     @showToolbar={{false}}
     @value={{stringify this.content}}
-    @options={{hash readOnly=true lineNumbers=false autoHeight=true gutters=false theme="hashi auto-height"}}
```

Comment:
all options objects were spread out so that the individual properties were sent to the json-editor.

Reply:
Good catch. I removed this because before it was used to set the viewportMargin. [See here](https://github.com/hashicorp/vault/pull/14659/files#diff-9401613f84fc7b84c9749d59349bcac5eee3d99028b28d2241b6e3e985dd419aL43)

Reply:
Ahh ok I see now thanks!